homeassistant/components/local_calendar/diagnostics.py

```diff
@@ -19,7 +19,7 @@ async def async_get_config_entry_diagnostics(
     payload: dict[str, Any] = {
         "now": dt_util.now().isoformat(),
         "timezone": str(dt_util.DEFAULT_TIME_ZONE),
-        "system_timezone": str(datetime.datetime.utcnow().astimezone().tzinfo),
+        "system_timezone": str(datetime.datetime.now().astimezone().tzinfo),
```

Comment:
I'm not sure this change is correct. I think we're after this documented behavior:

> If self is naive, it is presumed to represent time in the system timezone.

https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone

Reply:
Ok, good.